card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
+
```

Comment:
This is the one I'm having the most trouble with. It is used for the use case of adding cards to an existing app. First, should the command run under the 'rails' or 'wagn' command? I think it would be best if they are available from both, but I couldn't get that to work yet.

There are also more details to work out in terms of dependency on the gems. I think it best for dependencies on the wagn gem are only for development tasks, running tests, compiling assets, generating mods, etc. but I think we still have some assets in wagn that all rich_html based applications will need.